client/src/components/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Header() {
```

Comment:
I would pass here a prop, maybe call it `label` that holds the string, that is rendered in the return statement of the component. This way, the `Header` component is reusable. Maybe on other `pages`?